scylladb/sphinx-scylladb-theme
# Added  substring_removed theme options

Related issue https://github.com/scylladb/sphinx-scylladb-theme/issues/109

Adds two new ``theme_options``:

* ``branch_substring_removed``: Part of the branch name to skip. When the project does not set this option, the theme removes the prefix ``branch-`` for all releases listed in the multiversion dropdown.
* ``tag_substring_removed``: Part of the tag name to skip. When the project does not set this option, the theme removes the suffix``-scylla`` for all releases listed in the multiversion dropdown.

## How to test this PR

The theme's sample documentation project is building docs for ``master`` branch and ``stable`` tag. 

You could set``tag_substring_removed`` in``theme_options`` dict  (conf.py) to ``ble``.
Then, run ``make multiversionpreview`` and check if the dropdown rendered just displays ``sta`` (the part not skipped).

## Rolling out the changes
This addition does not introduce not breaking changes. After releasing a new theme version on PyPi, I'll review every project following a special tag/branch naming convention and submit a PR configuring these two variables.
